app/controllers/predicates/MandationStatusPredicate.scala

```diff
@@ -62,8 +62,7 @@ class MandationStatusPredicate @Inject()(mandationStatusService: MandationStatus
       case Right(status) =>
         Left(Redirect(user.uri).addingToSession(SessionKeys.mandationStatus -> status.mandationStatus))
       case Left(error) =>
-        logger.info(s"[MandationStatusPredicate][refine] - Error has been received $error")
-        logger.warn(s"[MandationStatusPredicate][refine] - Error has been received")
+        errorLog(s"[MandationStatusPredicate][refine] - Error has been received $error")
```

Comment:
same here

Reply:
i think this should be kept as errorLog since it is not able ti get the mandation status